api/machinemanager/machinemanager.go

```diff
@@ -19,10 +19,15 @@ type Client struct {
 	facade base.FacadeCaller
 }
 
+// MakeClient is a constructor function for a machine manager client
+func MakeClient(clientFacade base.ClientFacade, facadeCaller base.FacadeCaller) *Client {
```

Comment:
Is there a better name that we can use for this?
It is incongruent with the Go idioms new (for references) and make (for object values).

Reply:
That it is will fix. It just `NewClient` was already by the caller.